### PR TITLE
🔥 Remove CommandTreeRegistry

### PR DESCRIPTION
**Pull request by SPGoding**
It's unnecessary abstraction that obscures things. If we ever end up having multiple versions of the tree available at the same time, whatever package's responsible for that should manage all the trees internally.

### CODE REVIEW

File: __snapshots__/packages/language-server/test-out/util/toLS.spec.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      },
+      "end": {
+        "line": 2,
+        "character": 2
+      }
+    }
+  },
+  "insertTextFormat": 2,
+  "insertTextMode": 2
+}
```
Comment (TheAfroOfDoom) [inline __snapshots__/packages/language-server/test-out/util/toLS.spec.js:86]: looks like you need to `npm run clean` (or `clean:full`) maybe? guessing these snapshots shouldnt be showing up here

**Review by TheAfroOfDoom — APPROVED** [review]
curious what the initial motivation was behind storing a registry/map of trees?

also:

https://youtu.be/52LJaWDdG9c?t=26

File: __snapshots__/packages/language-server/test-out/util/toLS.spec.js (hunk 1 above)
Comment (TheAfroOfDoom) [reply __snapshots__/packages/language-server/test-out/util/toLS.spec.js:86]: i dont know why i felt the need to highlight the entire snapshot, sorry

Comment (SPGoding) [reply __snapshots__/packages/language-server/test-out/util/toLS.spec.js]: i even double checked to make sure i didn't include them :sob:

**Review by misode — APPROVED** [review]

**Pull request merged**